cloudformation.yaml

```diff
@@ -59,14 +63,22 @@ Resources:
               Action:
                 - lambda:InvokeFunction
               Resource: "*"
-        - PolicyName: dynamoTable
+        - PolicyName: unEncryptedDynamoTable
           PolicyDocument:
             Statement:
               Effect: Allow
               Action:
               - dynamodb:PutItem
               - dynamodb:GetItem
               Resource: !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/cas-auth-${Stage}'
+        - PolicyName: dynamoTable
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+              - dynamodb:PutItem
+              - dynamodb:GetItem
+              Resource: !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/daily-edition-trial-periods-${Stage}'
```

Comment:
I added the permission to use the new table even though the lambda will not use it yet,  when we switch the lambda we can remove the unEncryptedTable permissions